mozilla_version/test/test_fenix.py

```diff
@@ -0,0 +1,308 @@
+import pytest
+import re
+
+from distutils.version import StrictVersion, LooseVersion
+
+import mozilla_version.gecko
+
+from mozilla_version.errors import PatternNotMatchedError, TooManyTypesError, NoVersionTypeError
+from mozilla_version.fenix import FenixVersion
+from mozilla_version.test import does_not_raise
+
+
+VALID_VERSIONS = {
+    '0.1': 'release',
```

Comment:
of note, `0.1` was from official tags, but I can drop it for sure.

Reply:
Oh gosh, I thought they've always follow semver, but `0.1` is the only counter-example 😅

Some `GeckoVersion`s have a way to specify known edge cases: 

https://github.com/mozilla-releng/mozilla-version/blob/2180fe058bc3ada6fa8cbdd24cc847f27b418798/mozilla_version/gecko.py#L416-L429

It might be too big just capture a single case. I'd say: let's not support `0.1` for now and enhance `FenixVersion` to support this case later. For instance, `GeckoVersion` didn't support old version numbers until https://github.com/mozilla-releng/mozilla-version/pull/59